src/pages/components/modal/usage.mdx

```diff
@@ -330,7 +330,18 @@ There are three buttons in the progress modal footer: Cancel, Previous, Next. Ea
 
 ## Related
 
+#### Modal vs. notification
+
+Modals are highly disruptive and block tasks until dismissed by the user. A modal should only be used as a form of notification if it provides the user with critical information immediately related to their task. For non-critical messaging considering using a toast or inline notification instead of a modal. Toast and inline notifications provide users with non-disruptive feedback or the status of an action without breaking their workflow.
```

Comment:
```suggestion
Modals are highly disruptive and block tasks until dismissed by the user. A modal should only be used as a form of notification if it provides the user with critical information immediately related to their task. For non-critical messaging, consider using a toast or inline notification instead of a modal. Toast and inline notifications provide users with non-disruptive feedback or the status of an action without breaking their workflow.
```